src/components/shared/ProjectHeader/index.tsx

```diff
@@ -0,0 +1,136 @@
+import { t } from '@lingui/macro'
+
+import ProjectLogo from 'components/shared/ProjectLogo'
+import { ThemeContext } from 'contexts/themeContext'
+
+import { useContext } from 'react'
+
+import { BigNumber } from 'ethers'
+
+import { ProjectMetadataV3 } from 'models/project-metadata'
+
+import Paragraph from 'components/shared/Paragraph'
+
+import SocialLinks from './SocialLinks'
+
+export default function ProjectHeader({
+  projectId,
+  handle,
+  metadata,
+  isArchived,
+  actions,
```

Comment:
`actions` is the wrench, settings icon etc. I've created a V1-specific Actions component below.
Everything else in ProjectHeader can be re-used.